apps/GenderDetection/genderdetection.cpp

```diff
@@ -37,17 +37,15 @@ void signalCallbackHandler(int signum) {
 class GenderDetector {
 
 public:
-  std::string genderProto = "./gender_deploy.prototxt";
-  std::string genderModel = "./gender_net.caffemodel";
   cv::Scalar MODEL_MEAN_VALUES =
       cv::Scalar(78.4263377603, 87.7689143744, 114.895847746);
   std::vector<std::string> genderList = {"Male", "Female"};
 
   [[noreturn]] void compute(iceflow::RingBuffer<iceflow::Block> *input,
                             iceflow::RingBuffer<iceflow::Block> *output,
-                            int outputThreshold) {
+                            int outputThreshold, std::string ml_proto, std::string ml_model) {
```

Comment:
If I am not mistaken, it is better to use const string references here:

```suggestion
                            int outputThreshold, const std::string& ml_proto, const std::string& ml_model) {
```